Description: [PATCH] pam: always install pam file
 Packported from https://github.com/hyprwm/hyprlock/commit/d9a6229434fba475ea42b634ee2f03919236798d.patch
 .
 hyprlock (0.3.0-1ppa1) noble; urgency=medium
 .
   * Initial release. Follows release 0.3.0.
Author: Vaxry <vaxry@vaxry.net>

---
The information above should follow the Patch Tagging Guidelines, please
checkout https://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
Bug: <upstream-bugtracker-url>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: (no|not-needed|<patch-forwarded-url>)
Applied-Upstream: <version>, (<commit-url>|commit:<commid-id>)
Reviewed-By: <name and email of someone who approved/reviewed the patch>
Last-Update: 2024-05-31

--- hyprlock-0.3.0.orig/CMakeLists.txt
+++ hyprlock-0.3.0/CMakeLists.txt
@@ -83,9 +83,4 @@ protocol("unstable/linux-dmabuf/linux-dm
 # Installation
 install(TARGETS hyprlock)
 
-install(CODE "
-    if (NOT EXISTS \"${CMAKE_INSTALL_FULL_SYSCONFDIR}/pam.d/hyprlock\")
-        install(FILES \"${CMAKE_SOURCE_DIR}/pam/hyprlock\" DESTINATION \"${CMAKE_INSTALL_FULL_SYSCONFDIR}/pam.d\")
-    endif()
-")
-
+install(FILES ${CMAKE_SOURCE_DIR}/pam/hyprlock DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/pam.d)
